src/utils/FSHLogger.ts

```diff
@@ -92,3 +112,17 @@ class LoggerStats {
 }
 
 export const stats = new LoggerStats();
+
+export class ErrorsAndWarnings {
```

Comment:
Oooo I think this could help clean up how we access the errors and warnings in FSH Online ✨